Adding ability for op to see the inventory of chests with a block above it

Maxopoly opened the pull request:
Adressing https://github.com/Civcraft/Humbug/issues/58

@rourke750

rourke750 commented on src/main/java/com/untamedears/humbug/Humbug.java:
```diff
@@ -2511,6 +2513,35 @@ public void onSignLoads(ChunkLoadEvent event) {
+  
+  @EventHandler(priority = EventPriority.HIGHEST)
+  public void adminAccessBlockedChest(PlayerInteractEvent e) {
+	  if (!e.getPlayer().isOp()) {
```
also add a permission check here if you could as well.  Otherwise looks perfect.

rourke750 commented:
Also can you increment the version in the pom.

rourke750 commented:
ok to test

rourke750 commented:
@Maxopoly Did you test this?

Maxopoly commented:
I did, it didn't work, I fixed it here: https://github.com/Civcraft/Humbug/commit/7b059c7561b72ca4af7022ede9974f2d3aa343bb

I already tested it with that and it's working, I also changed it so you dont get a copy of the chest inventory, but instead get the chest inventory itself so you can actually take items out and put items in there.
